radiomics/firstorder.py

```diff
@@ -5,27 +5,31 @@
 
 class RadiomicsFirstOrder(base.RadiomicsFeaturesBase):
   r"""
-  First-order statistics describe the distribution of voxel intensities within the image region defined by the mask through commonly used and basic metrics.
+  First-order statistics describe the distribution of voxel intensities within the image region defined by the mask
+  through commonly used and basic metrics.
 
   Let:
 
   :math:`\textbf{X}` denote the three dimensional image matrix with :math:`N` voxels
 
   :math:`\textbf{P}(i)` the first order histogram with :math:`N_l` discrete intensity levels,
-  where :math:`l` is defined by the number of levels is calculated based on the binWidth parameter of the constructor.
+  where :math:`N_l` is defined by the number of levels, calculated based on the ``binWidth`` parameter.
 
   :math:`p(i)` be the normalized first order histogram and equal to :math:`\frac{\textbf{P}(i)}{\sum{\textbf{P}(i)}}`
 
-  Following addiotional settings are possible:
+  Following additional settings are possible:
 
-  - voxelArrayShift [2000]: This amount is added to the gray level intensity in Energy, Total Energy and RMS, this is to prevent negative values from occuring when using CT data.
+  - voxelArrayShift [0]: Integer, This amount is added to the gray level intensity in features Energy, Total Energy and
+    RMS, this is to prevent negative values. __If using CT data, or data normalized with mean 0, consider setting this
+    parameter to a fixed value that ensures non-negative numbers in the image. Bear in mind however, that the larger the
+    value, the larger the volume confounding effect will be.__
```

Comment:
I added the suggestion at the definition of `voxelArrayShift` parameter in the documentation (at the start of firstorder feature class)